pyveda/legacy/datapoint.py

```diff
@@ -63,8 +63,19 @@ def y(self):
     def image(self):
         """ Returns a delayed dask call for fetching the image for a data point """
         token = gbdx.gbdx_connection.access_token
-        load = load_image(self.links["image"]["href"], token, self.imshape, dtype=self.dtype)
-        return da.from_delayed(load, shape=self.imshape, dtype=self.dtype)
+        delay = load_image(self.links["image"]["href"], token, self.imshape, dtype=self.dtype)
+        return self._delayed_image(delay)
+
+    @property
+    def thumbnail(self):
+        """ Returns a delayed dask call for fetching a thumbnail for a data point """
```

Comment:
Do we need to use dask for this? We just recently moved away from this dependency, which is an annoying one for breaking changes, so we should really try to only use it if it's crucial to make dask task graphs for large compute or something like taht